### DISCUSSION TITLE
Build standalone library linked with zephyr

### DISCUSSION THREAD

**Discussion by paulinemartinez** [zephyrproject-rtos/zephyr#43026] [Q&A]
Hi,
Maybe I missed the information somewhere, but I could not find a way to build a library, linked with zephyr (because using some zephyr features such as network buffers) without building an App and its firmware.
My lib is external (sources outside zephyr tree) and I build it as an external module.
I manage to build it by create a "fake app" doing nothing, and with which I link the lib. I can find the static library in the build directory.
But it is not very clean and I would prefer to be able to build ONLY the library.
Is it possible?
Thanks in advance your your help!

**Comment by yashi** [comment]
The Zephyr build system currently does NOT support building a library.
